lib/errors/error.messages.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
   APPLICATIONID_MISSING: 'ApplicationId not configured.',
   AUTHZV2_CONFIG_MISSING: 'Authzv2 not configured.',
+  DISABLED_CONFIGURATION: 'Configuration is disabled, everything will be allowed. Received params: ',
```

Comment:
When this is logged, can you tell it is from the authz library?

Reply:
Updated the error message to make this more clear